src/items/class-features/bedside_manner_(ex).json

```diff
@@ -0,0 +1,115 @@
+{
+  "_id": "TnDwezeK41boR3kD",
+  "name": "Bedside Manner (Ex)",
+  "type": "feat",
+  "img": "icons/svg/mystery-man.svg",
+  "system": {
+    "type": "",
+    "ability": "",
+    "abilityMods": {
+      "parts": []
+    },
+    "actionTarget": "",
+    "actionType": "",
+    "activation": {
+      "type": "none",
```

Comment:
This doesn't need to be a "none" activation, you can just set it to blank

Reply:
Cleared it.